singer-io/tap-github
# TDL-5961 Support of custom domain

# Description of change
- Added support of custom domain.
- Added unit test case for the changes.

**Note**: Build is failing because config parameter `base_url` is missing.
# Manual QA steps
 - Verify that if a custom domain is provided in the config then that domain is being used by tap to call the REST API.
 - Verify that if a custom domain is not provided in the config then that default domain is being used by tap to call the REST API.
 
# Risks
 - 
 
# Rollback steps
 - revert this branch
